draft-ietf-idr-wide-bgp-communities.xml

```diff
@@ -983,20 +983,27 @@ values 100..104 may be used for this locally defined Wide Community.</t>
 
 <t>Parameter semantics:</t>
 <t><list>
-<t>The parameter TLV shall consist of exactly one Unsigned Integer32 Atom value that is
-constrained to have a value of 2..8.</t>
+<t>The parameter TLV, used to represent the number of AS_PATH
+prepends that will be added by this community, shall consist of
+exactly one Unsigned Integer32 Atom value that is constrained to have a value of
+2..8.</t>
 </list></t>
 
 </section>
 
 <section title="Example Type 1 BGP Wide Community Encoding">
 
-<t>AS_PATH prepend 4 TIMES TO AS 2424, AS 8888, to peers marked as
-  Amsterdam (100) or to peers marked Moscow (104), but not to peers 
-  in New York (101).</t>
+<t>In this example, the BGP Wide Community defined above is used by
+a BGP Speaker to AS_PATH prepend BGP routes containing this
+community AS_PATH prepend 4 TIMES when this route is to be
+distribute to any of AS 2424, AS 8888, to peers marked as
+Amsterdam (100) or to peers marked Moscow (104). However, such
+prepending would not be done to peers that have been configured
+in the User Class of , but not to peers of New York (101),

```

Comment:
Not sure if we need to state "User Class of  ,"  

of .... what ? 

And the comma has an extra space there